docs/RationaleHW.md

```diff
@@ -65,6 +67,13 @@ operations.  Instead, it is designed to be a flexible and extensible substrate
 that may be extended with higher level dialects mixed into it (like `sv`,
 `comb`, `seq`, etc).
 
+_MW: Above is the first mention of `seq` dialect_
+
+_MW:  My impression is that the `hw` dialect is modeling the concept of components
+that can be plugged together with "wires", without dealing with any of the functionality
+within the components. Or is connection not part of `hw`? If it is I think it's worth calling out
+specifically here._
```

Comment:
Good point, connect semantics are actually the domain of the 'sv' dialect.  I'll mention that.